lib/logstash/outputs/elasticsearch.rb

```diff
@@ -10,44 +10,32 @@
 require "uri" # for escaping user input
 require "logstash/outputs/elasticsearch/http_client"
 
-# This output lets you store logs in Elasticsearch and is the most recommended
-# output for Logstash. If you plan on using the Kibana web interface, you'll
-# want to use this output.
+# This plugin is the recommended method of storing logs in Elasticsearch.
+# If you plan on using the Kibana web interface, you'll want to use this output.
 #
-# This output only speaks the HTTP, which is the preferred protocol for interacting with Elasticsearch. By default
-# Elasticsearch exposes HTTP on port 9200.
-#
-# We strongly encourage the use of HTTP over the node protocol. It is just as
-# fast and far easier to administer. For those wishing to use the java protocol please see the 'elasticsearch_java' gem.
+# This output only speaks the HTTP protocol. HTTP is the preferred protocol for interacting with Elasticsearch as of Logstash 2.0.
+# We strongly encourage the use of HTTP over the node protocol for a number of reasons. HTTP is only marginally slower,
+# yet far easier to administer and work with. When using the HTTP protocol one may upgrade Elasticsearch versions without having
+# to upgrade Logstash in lock-step. For those wishing to use the java protocol please see the 'elasticsearch_java' gem.
```

Comment:
Call it "node protocol" here maybe? This is the first mention of "java protocol" I think.